requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-pytest>=5.4.1
-pytest-flask-sqlalchemy>=1.0.2
+pytest==5.4.1
```

Comment:
I'd just as soon keep the `>=` for the testing libraries. I'm fairly certain I'll always want the updates, and I'm not concerned at all about the testing requirements breaking my functioning app.